runtime/tests/port/j9strTest.c

```diff
@@ -1528,46 +1528,56 @@ j9str_WinacpToMutf8(struct J9PortLibrary *portLibrary)
 #if defined(J9ZOS390)
 /**
  * Check whether a null/non-null terminated string is properly handled by fstring().
+ * Also checks if number, long number, long long number, and float are properly handled.
```

Comment:
This function participates in those checks, but I don't think it appropriate to claim they're done here.

Reply:
removed this comment and the one above to the actual test function